addons/web/static/src/js/views/kanban/kanban_model.js

```diff
@@ -201,9 +201,10 @@ var KanbanModel = BasicModel.extend({
     loadRecords(groupID, options = { loadMore: false, domainExtension: [] }) {
         const group = this.localData[groupID];
         group.domainExtension = options.domainExtension || group.domainExtension;
-        return this.reload(group.id, {
-            loadMoreOffset: options.loadMore ? group.loadMoreOffset + group.limit : 0
-        });
+        const loadMoreOffset = options.loadMore ? group.data.length : 0;
+        const highestParent = this._getHighestParent(group);
+        group.limit = highestParent.limit;
+        return this.reload(group.id, { loadMoreOffset });
```

Comment:
double check too :-)

Reply:
OK so definitely create a recursive function to get the highest parent in a chain of datapoints is not at all a nice idea.
I could have just made a simple loop.
Furthermore, I wrote those lines because of a specific problem: when the basic model performs a readgroup, it may replace the limit of a list datapoint such that all previously loaded records (e.g. if we are coming back to the kanban view from a form view) are reloaded. This may be a weird way to do so but it works... while bringing the side effect that further « load more » search_read will use the new limit and thus retrieve an higher amount of records than the initial « arch view limit ».

IMHO keeping this side effect can be tolerable and we should just keep it in mind when we will refactor the kanban view.

Reply:
Or, I could keep resetting the datapoint limit attribute in the kanban_model just before loading records but doing it this way:
```suggestion
        const loadMoreOffset = options.loadMore ? group.data.length : 0;
        group.limit = this.loadParams.limit;
        return this.reload(group.id, { loadMoreOffset });
```